src/main.cpp

```diff
@@ -407,6 +407,9 @@ void setup()
     Wire.begin();
     i2cScanner->scanPort(ScanI2C::I2CPort::WIRE);
 #elif defined(I2C_SDA) && !defined(ARCH_RP2040)
+#if defined(I2C_PWR_ENABLE)
```

Comment:
Looking through the source i have a better suggestion. Use PIN_POWER_EN instead. Right now the use is limited to the T-ECHO device but it's the only variant that actually uses that define. so you can remove the limitation to T-ECHO in Line 244 and use that instead.

Reply:
Thank you, it actually works properly using PIN_POWER_EN, my implementation did not set the GPIO direction correctly anyway. After ungating PIN_POWER_EN and using that code path, the SHTC3 clone gets detected fine in the environment telemetry module.